MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Selections in MAUI Calendar widget | Syncfusion
+description: Learn here all about Selections feature in Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Selections in MAUI Calendar (SfCalendar)
+The Selection mode is specified in the calendar property enumeration. Can select the dates by tap the cell in the calendar. `SfCalendar` provides three types mode such as `Single`, `Multiple` and `Range` Selection. The default `SelectionMode` is Single that allows the user to select one date at a time.
+
+>**NOTE** Can able to select the cells in the Year, Decade and Century views in the calendar only when the `AllowViewNavigation` property is set to false.
+
+## Single selection
+The `Single` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Single`. In this Selection, you can able to select a single date from the cell.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Single">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Single;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view single selection in .NET MAUI Calendar.](images/selection/maui-month-view-single-selection.png)
+{% endcolumn %}
+
+>**NOTE**
+* In the Year, Decade and Century views can able to select the cells, only when the `AllowViewNavigation` property is set to false.
+* In this scenario, the `selection changed` callback will return the first date of the month, year, decade or century of the selected cell.
+Eg: 
+* In the year view, when the Dec month cell is selected then the selected date value will be 01-12-2022.
+* In the decade view, when the (2022) year cell is selected then the selected date value will be 01-01-2022.
+* In the century view, when the (2020-2029) decade cell is selected then the selected date value will be 01-01-2020.
+
+
+## Multiple selection
+The `Multiple` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Multiple`. In this Selection, you can able to select a multiple dates from the cell. If you want to remove the selected cell by clicking the same cell again.
```

Comment:
The `Multiple` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Multiple`. In this selection, you can select multiple dates from the cell. If you want to remove the selected cell, click the same cell again.

Reply:
Content modified